content/en/user-guide/tools/localstack-desktop/index.md

```diff
@@ -0,0 +1,52 @@
+---
+title: "LocalStack Desktop"
+weight: 8
+description: >
+  Getting started with LocalStack Desktop application
+tags: ["desktop"]
+aliases:
+  - /tools/localstack-desktop/
+  - /user-guide/tools/localstack-desktop/
+---
+
+LocalStack Desktop is a desktop client that allows users to easily control and interact their LocalStack instance. With LocalStack Desktop, users can start and stop their LocalStack instance with a single click, create a new container, view logs, interact with LocalStack container via cli and use our resource browser.
```

Comment:
Is it only available to Pro/Team users? Or generally available?

Reply:
Is available for everybody but you need to log in to use the resource browser